docs/extensibility/README.md

```diff
@@ -8,8 +8,8 @@ Extensibility in Busola is a feature that allows you to create a separate dedica
 
 To automatically add the UI page for your CRD, follow these steps:
 
-1.  In the **Cluster Details** view, go to **Configuration > Custom Resource Definitions** and click on your CRD.
```

Comment:
There's no `Extensions` in `Cluster Details` page. We should go with `Configuration` -> `Extensions`.
<img width="1196" alt="image" src="https://user-images.githubusercontent.com/10504661/182552022-402d60fe-a0f4-4c59-a568-113476a1e079.png">